tests/regression/13-privatized/89-write-lacking-precision.c

```diff
@@ -0,0 +1,30 @@
+// PARAM: --set ana.base.privatization write
+#include<pthread.h>
+struct a {
+  char* b;
+};
+
+struct a *c;
+struct a h = {""};
+struct a i = {"string"};
+
+void* d(void* args) {
+  struct a r;
```

Comment:
This seems unused. Is it somehow necessary for the issue to trigger?

Reply:
I removed it.